docs: remove reference to production/kubernetes (#6079)
grafana/agent: docs/sources/static/set-up/install/install-agent-kubernetes.md

# Deploy Grafana Agent in static mode on Kubernetes

> Grafana Agent in static mode can be deployed on Kubernetes by using the Helm chart for Grafana Agent.

Reviewer: ```suggestion
You can use the Helm chart for Grafana Agent to deploy Grafana Agent in static mode on Kubernetes.
```
